### PR TITLE
Edits to doc comments in the EntryPoints files

### PR DESCRIPTION
**Pull request by chuckdude**
Edits to the documentation comments (`///` lines) in the `.Swift` API source files in the `EntryPoints` directory.

### Motivation:

Editing for style.

### Modifications:

Edits to the text in some `///` comments throughout.

### Result:

Documentation produced from the source files will generate and conform to style.

### Checklist:

- [x] Code and documentation should follow the style of the [Style Guide](https://github.com/apple/swift-testing/blob/main/Documentation/StyleGuide.md).
- [x] If public symbols are renamed or modified, DocC references should be updated.

### CODE REVIEW

File: Sources/Testing/EntryPoints/ABIEntryPoint.swift [hunk 1]
```diff
@@ -27,7 +27,7 @@
 /// and then invokes available tests in the current process.
 ///
 /// - Warning: This function's signature and the structure of its JSON inputs
-///   and outputs have not been finalized yet.
+///   and outputs have not been finalized yet.@Comment{ QUERY: Has this been finalized? If so, can we remove the Warning? If not, we should make this a double-slash comment for initial release. }
```
Comment (grynspan) [inline Sources/Testing/EntryPoints/ABIEntryPoint.swift:30]: Not yet. See #368. (Also `@Comment` isn't valid here.)

File: Sources/Testing/EntryPoints/ABIEntryPoint.swift [hunk 2]
```diff
@@ -13,10 +13,10 @@
 /// to remain version-agnostic regarding the testing library.
 ///
 /// - Parameters:
-///   - argumentsJSON: A buffer to memory representing the JSON encoding of an
-///     instance of `__CommandLineArguments_v0`. If `nil`, a new instance is
-///     created from the command-line arguments to the current process.
-///   - eventHandler: An event handler to which is passed a buffer to memory
+///   - argumentsJSON: A buffer to memory that represents the JSON encoding of an
+///     instance of `__CommandLineArguments_v0`. If `nil`, and creates a new instance
```
Comment (grynspan) [inline Sources/Testing/EntryPoints/ABIEntryPoint.swift:17]: This is grammatically incorrect.

File: Sources/Testing/EntryPoints/EntryPoint.swift [hunk 3]
```diff
@@ -348,7 +348,7 @@ public func configurationForEntryPoint(from args: __CommandLineArguments_v0) thr
 ///   - path: The path to which events should be streamed. This file will be
 ///     opened for writing.
 ///
-/// - Throws: Any error that occurs opening `path`. Once `path` is opened,
+/// - Throws: Any error that occurs opening `path`. After opening the `path`,
```
Comment (grynspan) [inline Sources/Testing/EntryPoints/EntryPoint.swift:351]:
```suggestion
/// - Throws: Any error that occurs opening `path`. After opening `path`,
```

File: Sources/Testing/EntryPoints/EntryPoint.swift [hunk 4]
```diff
@@ -361,7 +361,7 @@ public func configurationForEntryPoint(from args: __CommandLineArguments_v0) thr
 /// objects are guaranteed not to contain any ASCII newline characters (`"\r"`
 /// or `"\n"`) themselves.
 ///
-/// The file at `path` can be a regular file, however to allow for streaming a
+/// The file at `path` can be a regular file, however, to allow for streaming a
```
Comment (grynspan) [inline Sources/Testing/EntryPoints/EntryPoint.swift:364]: This change is incorrect.

File: Sources/Testing/EntryPoints/EntryPoint.swift [hunk 5]
```diff
@@ -452,17 +452,17 @@ extension Event.ConsoleOutputRecorder.Options {
     return result
   }
 
-  /// Whether or not the current process's standard error stream is capable of
-  /// accepting and rendering ANSI escape codes.
+  /// A Boolean value that indicates whether the current process's standard
```
Comment (grynspan) [inline Sources/Testing/EntryPoints/EntryPoint.swift:455]: This is a function, not a boolean value, so it should probably be "Verb whether or not blah blah".

File: Sources/Testing/EntryPoints/EntryPoint.swift [hunk 6]
```diff
@@ -452,17 +452,17 @@ extension Event.ConsoleOutputRecorder.Options {
...
     // Determine if this file handle appears to write to a Terminal window
     // capable of accepting ANSI escape codes.
     if fileHandle.isTTY {
       return true
     }
 
-    // If the file handle is a pipe, assume the other end is using it to forward
-    // output from this process to its own stderr file. This is how `swift test`
+    // If the file handle is a pipe, assume the other end uses it to forward
+    // output from this process to its own `stderr` file. This is how `swift test`
```
Comment (grynspan) [inline Sources/Testing/EntryPoints/EntryPoint.swift:465]: This is a source-level comment, not documentation.

File: Sources/Testing/EntryPoints/SwiftPMEntryPoint.swift [hunk 7]
```diff
@@ -23,7 +23,7 @@ private import TestingInternals
 /// This function examines the command-line arguments represented by `args` and
 /// then invokes available tests in the current process.
 ///
-/// - Warning: This function is used by Swift Package Manager. Do not call it
+/// - Warning: This function is used by Swift Package Manager. Don't call it
```
Comment (grynspan) [inline Sources/Testing/EntryPoints/SwiftPMEntryPoint.swift:26]: Symbols with two leading underscores aren't published in our DocC bundle, so no need to revise them.

File: Sources/Testing/EntryPoints/XCTestScaffold.swift [hunk 8]
```diff
@@ -89,8 +89,8 @@ public enum XCTestScaffold: Sendable {
   ///   - testCase: An `XCTestCase` instance that hosts tests implemented using
   ///     the testing library.
   ///
-  /// Output from the testing library is written to the standard error stream.
-  /// The format of the output is not meant to be machine-readable and is
+  /// Writes output from the testing library to the standard error stream.
```
[the remark this answers is not in the document]
Comment (grynspan) [reply Sources/Testing/EntryPoints/XCTestScaffold.swift:92]: Regardless, `XCTestScaffold` is going to be removed; you don't need to worry about proofreading it.

**Comment by chuckdude** [comment]
Per Jonathan; these edits aren't needed here, so I'm closing the PR without merging.

**Pull request closed without merging**